graph_weather/__init__.py

```diff
@@ -1,4 +1,5 @@
 """Main import for the complete models"""
 
+from .data.nnjai_wrapp import SensorDataset, collate_fn
```

Comment:
I don't think we should actually import this here, since we have `nnja` as an optional dependency. I think to import this dataset, like in the tests, we should just do `from graph_weather.data.nnja_ai import SensorDataset` for example. I also renamed this on the main branch, but could you update the `nnjai_wrapp.py` to be named `nnja_ai` as I think that is maybe more consistent naming.